autogen/agentchat/contrib/agent_eval/criterion.py

```diff
@@ -0,0 +1,69 @@
+import json
+from typing import List
+
+
+class Criterion:
+    """
+    A class that represents a criterion for agent evaluation.
+    """
+
+    def __init__(self, name: str, description: str, accepted_values: List[str], sub_criteria=[]):
```

Comment:
One weird thing about Python is that if you do something like `sub_criteria=[]`, this list will be globally defined. let's say you have another function argument called `sub_criteria_2=[]`, then both subcriterias will share the same array. To avoid any weird accidental list mutation, we should do this:
 ```suggestion
     def __init__(self, name: str, description: str, accepted_values: List[str], sub_criteria: List[Criterion] = list()):
```

Reply:
That's good to know, thanks. Updated to `sub_criteria=list()` as trying to use the Criterion typing throws a "name 'Criterion' not defined" error (if there's a way around this let me know)

Reply:
```python
from __future__ import annotations
```
Should solve the problem